content/articles/2019-04-22-dplyr-0-8-1.Rmarkdown

```diff
@@ -0,0 +1,98 @@
+---
+title: dplyr 0.8.1
+slug: dplyr-0-8-1
+author: Romain François
+description: >
+    dplyr 0.8.1 is now on CRAN.
+date: '2019-04-22'
+categories: [package]
+tags:
+  - dplyr
+photo:
+  url: https://unsplash.com/photos/BiZ-_6kNjbI
+  author: Sophie Elvis
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+# options(crayon.enabled = TRUE)
+```
+
+## Introduction
+
+We're delighted to announce the release of dplyr 0.8.1 on CRAN `r emo::ji("tada")` !
+
+This is a minor release that address follow ups from the community after the 
+release of the [0.8.0 version](https://www.tidyverse.org/articles/2019/02/dplyr-0-8-0/). 
+
+## group_map() and group_modify()
+
+Shortly after the release of 0.8.0, we were notified by several members of the 
+community that `group_map()` was great, except it didn't do what they had expected `r emo::ji("grimacing")`. 
+
+Because the function was (and still is) marked as experimental, we allowed ourselves to 
+rectify the situation: 
+
+ - The behaviour we previously had was renamed `group_modify()`.
```

Comment:
I recommend revealing the connection to `purrr::modify()` to help people see the bigger picture / connections here.

Basically, I'd have at least one very pointed sentence about renaming to better fit with existing expectations about the return value.

Reply:
Thanks. I've added some updates.